docs/slack.rst

```diff
@@ -33,6 +33,10 @@ like ours, but many people seem to feel a freedom in knowing that their words wi
 We understand that not everyone loves this decision,
 but it's the way things are currently set up and we don't plan to change it in the near future. 
 
+That doesn't mean everything gets lost, though - you can always react to a message with the 
+``:suggest-for-newsletter:`` emoji to nominate it for inclusion in our next monthly newsletter. 
+Sign up and see the newsletter archives at :doc:`/newsletter`! 
```

Comment:
```suggestion
Read the newsletter archives at :doc:`/newsletter` and subscribe if you want them directly in your inbox! 
```

otherwise it reads a bit like you need to subscribe to read to me